### PR TITLE
Multipolygon support

### PR DESCRIPTION
**Pull request by mclaeysb**
- Transform: MultiPolygon support
- Transform: Small readme correction on handedness

### CODE REVIEW

File: packages/transform/README.md [hunk 1]
```diff
@@ -197,33 +197,33 @@ const transformedPolygonGeoJSON = transformer.transformForwardAsGeojson(
...
+| <img width="100" src="../ui/src/lib/shared/images/transformations/projective.svg" alt="projective">             | `projective`                               | Projective or 'perspective' transformation, used for aerial images       | Follow perspective rules. Preserves lines and cross-ratios.                                                                          | 4                      |
 
 ### Transformer methods
 
 Once a transformer is built, it can be used to transform geometries forward and backward.
 
-All transformer methods accepts points, line strings as well as polygons, both as simple geometries or GeoJSON geometries. There are, however, separate methods for transforming to simple geometries or to GeoJSON geometries. There are also separate methods for transforming forward or backward.
+All transformer methods accepts points, lineStrings as well as polygons (and multiPoints, multiLineStrings and multiPolygons), both as simple geometries or GeoJSON geometries. There are, however, separate methods for transforming to simple geometries or to GeoJSON geometries. There are also separate methods for transforming forward or backward.
```
Comment (bertspaan) [inline packages/transform/README.md:214]: GeoJSON objects are capitalized; Points, LineStrings, etc.

Comment (mclaeysb) [reply packages/transform/README.md]: Fixed this!

**Comment by mclaeysb** [comment]
FYI: Not ready to merge yet!

**Comment by mclaeysb** [comment]
This is ready for merger!

**Comment by mclaeysb** [comment]
The last commit is the results of some tinkering after discussion. Seems a lot faster and more efficient like this!

Let's discuss this next time :)

**Comment by bertspaan** [comment]
Can I merge this PR?

**Comment by mclaeysb** [comment]
Yes, this is finished and I've tested it on a handful of advanced maps.

**Pull request merged**